packages/ionic-angular/README.md

```diff
@@ -4,31 +4,8 @@
 
 An Nx plugin for developing [Ionic](https://ionicframework.com/docs) React applications and libraries.
 
-For more complete documentation, visit [nxtend.dev](https://nxtend.dev/docs/ionic-angular/overview).
-
-## Table of Contents
-
-- [Usage](#usage)
-- [Maintainers](#maintainers)
-- [Contributing](#contributing)
-- [License](#license)
+For more complete documentation, visit [nxext.dev](https://nxext.dev/docs/ionic-angular/overview.html).
 
 ## Usage
 
-Visit the official [Getting Started](https://nxtend.dev/docs/ionic-angular/getting-started) documentation.
-
-## Maintainers
-
-[@devinshoemaker](https://github.com/devinshoemaker)
-
-## Contributing
-
-See [the contributing file](../../contributing.md)!
-
-PRs accepted.
-
-Small note: If editing the README, please conform to the [standard-readme](https://github.com/RichardLitt/standard-readme) specification.
-
-## License
-
-MIT © 2020 Devin Shoemaker
```

Comment:
@devinshoemaker Since this project has moved to another repo with another license, I think we can remove this one, but I'm not sure. What do you think?

Reply:
Not really a new License. Both are MIT licensed